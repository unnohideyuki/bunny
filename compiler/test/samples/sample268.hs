as = [True, True, True]
bs = [False, True, False]
cs = [False, False, False]

main = do print $ and as
          print $ or  as
          print $ and bs
          print $ or  bs
          print $ and cs
          print $ or  cs

