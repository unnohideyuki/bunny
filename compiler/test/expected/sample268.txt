True
True
False
True
False
False
